beast/tools/tests/test_write_sbatch_file.py

```diff
@@ -20,6 +20,7 @@ def test_sbatch_file():
 
     file = open(temp_file.name)
     content = file.read()
+    file.close()
```

Comment:
Another place where `with open() as ...` may or may not be useful.

Reply:
I'm ok with this for now.